### ISSUE TITLE
"make" not working with docker-compose V2 enabled

### ISSUE THREAD

**Issue by pautri** [Islandora-Devops/isle-dc#184]
if the "Use Docker Compose V2 release candidate" experimental feature is enabled (Docker Desktop 4.0.1 Mac Intel), make demo/local fail almost immediately and only a 0 byte docker-compose.yml file is written. I don't recall enabling this myself, it seems to be turned on by default now (see https://github.com/docker/for-mac/issues/5800#issuecomment-896082487)
```
Generating Secrets
Generated or copied ACTIVEMQ_PASSWORD!
Generated or copied ACTIVEMQ_WEB_ADMIN_PASSWORD!
Generated or copied ALPACA_ACTIVEMQ_PASSWORD!
Generated or copied ALPACA_KARAF_ADMIN_PASSWORD!
Generated or copied CODE_SERVER_PASSWORD!
Generated or copied DB_ROOT_PASSWORD!
Generated or copied DRUPAL_DEFAULT_ACCOUNT_PASSWORD!
Generated or copied DRUPAL_DEFAULT_CONFIGDIR!
Generated or copied DRUPAL_DEFAULT_DB_PASSWORD!
Generated or copied DRUPAL_DEFAULT_SALT!
Generated or copied FCREPO_DB_PASSWORD!
Generated or copied GEMINI_DB_PASSWORD!
Generated or copied JWT_ADMIN_TOKEN!
Generated or copied JWT_PRIVATE_KEY!
Generated or copied JWT_PUBLIC_KEY!
Generated or copied MATOMO_DB_PASSWORD!
Generated or copied MATOMO_USER_PASS!
Generated or copied TOMCAT_ADMIN_PASSWORD!
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name
WARN[0000] network gateway: network.external.name is deprecated in favor of network.name
1 error(s) decoding:

* error decoding 'Command': invalid command line string
make[1]: *** [docker-compose.yml] Error 15
make: *** [demo] Error 2
```

**Comment by DonRichards** [comment]
Same here. I can confirm this is the issue. By disabling 2.0 the build starts working. This needs to be fix before the next Docker Desktop release. They are planning to remove the revert docker-compose version option.
<img width="1220" alt="Screen Shot 2021-10-11 at 10 24 44 AM" src="https://user-images.githubusercontent.com/2738244/136807327-18108eb8-833f-41cd-823f-f89a49c223b0.png">

**Comment by DonRichards** [comment]
A different error related to this:
```shell 
1 error(s) decoding:

* error decoding 'Command': invalid command line string
make[1]: *** [Makefile:83: docker-compose.yml] Error 15
make[1]: Leaving directory '/home/user_name/isle-dc'
make: *** [Makefile:306: demo] Error 2
```

### To fix this within Linux
This is a versioning issue. As the title of this ticket suggests, there is an issue with running the latest docker-compose version (2.x). This is how to roll it back to the latest major version 1.2x.x
```shell
$ sudo curl -L https://github.com/docker/compose/releases/download/$(curl --silent "https://api.github.com/repos/docker/compose/tags" | grep 'name.*"1.2' | sort -r | head -1 | awk '{print $2}' | cut -d '"' -f2)/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
$ sudo chmod +x /usr/local/bin/docker-compose
```

**Comment by nigelgbanks** [comment]
Duplicate issue: https://github.com/Islandora/documentation/issues/1892

Pull request to address the issue: https://github.com/Islandora-Devops/isle-dc/pull/198

**Comment by nigelgbanks** [comment]
Resolved by #198
